src/metaphlan.jl

```diff
@@ -43,13 +43,19 @@ end
 
 
 """
-Option1: take a path to merged table (eg test/files/metaphlan_multi_profile.tsv)
+Option1: take a path to merged table (eg test/files/metaphlan_multi_test.tsv)
     and make CommunityProfile
 
-Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1_profile.tsv", "test/files/metaphlan_single2_profile.tsv"])
+Option2: take vector of paths to single tables (eg ["test/files/metaphlan_single1.tsv", "test/files/metaphlan_single2.tsv"])
```

Comment:
I actually think we should support both of these options as separate methods. There's a new [`commjoin`](https://biojulia.net/Microbiome.jl/latest/profiles/#Microbiome.commjoin) method that can be used for option 2 here.

Reply:
Yes absolutely! I just had not gotten to writing the method for Option 2. I just wrote the method using `commjoin`!